host-configs/lc-builds/corona_sycl.cmake

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
This header section should be replaced with this so it's consistent with other files:

###############################################################################
# Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
# and RAJA project contributors. See the RAJA/LICENSE file for details.
#
# SPDX-License-Identifier: (BSD-3-Clause)
###############################################################################